fabric-tool-attribute-api-v1/build.gradle

```diff
@@ -1,9 +1,10 @@
 archivesBaseName = "fabric-tool-attribute-api-v1"
-version = getSubprojectVersion(project, "1.2.3")
+version = getSubprojectVersion(project, "1.3.0")
 
 dependencies {
-    compile project(path: ':fabric-api-base', configuration: 'dev')
-    compile project(path: ':fabric-tag-extensions-v0', configuration: 'dev')
-    testmodCompile project(path: ':fabric-object-builder-api-v1', configuration: 'dev')
-	testmodCompile project(path: ':fabric-events-lifecycle-v0', configuration: 'dev')
+	compile project(path: ':fabric-api-base', configuration: 'dev')
+	compile project(path: ':fabric-resource-loader-v0', configuration: 'dev')
+	compile project(path: ':fabric-tag-extensions-v0', configuration: 'dev')
+	testmodCompile project(path: ':fabric-object-builder-api-v1', configuration: 'dev')
```

Comment:
Tool attributes depends on object builder and object builder depends on tool attributes.

Isn't this effectively a recursive dependency?

Reply:
Well I am not sure, and I didn't actually add this line, all I did is reformat this file, also this PR bases on #1073 and this line was changed there as well